master/docs/relnotes/index.rst

```diff
@@ -1,117 +1,132 @@
-Release Notes
-~~~~~~~~~~~~~
+Release Notes for Buildbot ``|version|``
```

Comment:
Yes, this PR needs lots of rework since I initially made it half a year ago. I will work on this.